zeus.go

```diff
@@ -185,40 +186,52 @@ func buildUrl(urls ...string) string {
 	return strings.Join(urls, "/") + "/"
 }
 
+func (zeus *Zeus) bucket(organization_and_bucket string) *Zeus {
+	zeus.OrganizationAndBucket = organization_and_bucket
+	return zeus
+}
+
 func (zeus *Zeus) request(method, urlStr string, data *url.Values) (
-body []byte, status int, err error) {
+	responseBody []byte, responseStatus int, err error) {
+	if zeus.OrganizationAndBucket == "" {
+		fmt.Println("Error: Please set bucket's name. ex) zeus.bucket(\"org1/bucket1\").GetLogs()")
+		return []byte{}, 0, err
+	}
+
 	if data == nil {
 		data = &url.Values{}
 	}
-	var resp *http.Response
-	if method == "POST" {
-		resp, err = http.PostForm(urlStr, *data)
-	} else if method == "GET" {
-		resp, err = http.Get(urlStr + "?" + data.Encode())
+	body := strings.NewReader(data.Encode())
+
+	var request *http.Request
+	if method == "GET" {
+		request, err = http.NewRequest("GET", urlStr+"?"+data.Encode(), nil)
+	} else if method == "POST" {
+		request, err = http.NewRequest("POST", urlStr, body)
 	} else if method == "PUT" {
-		req, err := http.NewRequest("PUT", urlStr, strings.NewReader(data.Encode()))
-		if err != nil {
-			return []byte{}, 0, err
-		}
-		req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
-		resp, err = http.DefaultClient.Do(req)
+		request, err = http.NewRequest("PUT", urlStr, body)
+		request.Header.Add("Content-Type", "application/x-www-form-urlencoded")
 	} else if method == "DELETE" {
-		req, err := http.NewRequest("DELETE", urlStr, strings.NewReader(data.Encode()))
-		if err != nil {
-			return []byte{}, 0, err
-		}
-		resp, err = http.DefaultClient.Do(req)
+		request, err = http.NewRequest("DELETE", urlStr, body)
 	}
 	if err != nil {
 		return []byte{}, 0, err
 	}
+	request.Header.Add("Authorization", "Bearer "+zeus.Token)
+	request.Header.Add("Bucket-Name", zeus.OrganizationAndBucket)
```

Comment:
@travelist 
Isn't enough?
https://github.com/CiscoZeus/go-zeusclient/pull/21/files#diff-e5c1a3b943c1c3c19e6d4624f505427bR196

I thought that new go-client don't need to deal with external_token, is this correct?